pkg/driver/node.go

```diff
@@ -73,12 +93,41 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 	// TODO when CreateVolume is implemented, it must use the same key names
 	subpath := "/"
 	encryptInTransit := true
+	iamAuth := false
 	volContext := req.GetVolumeContext()
+
 	for k, v := range volContext {
+
 		switch strings.ToLower(k) {
+		case "csi.storage.k8s.io/pod.name":
+			continue
+		case "csi.storage.k8s.io/pod.namespace":
+			continue
+		case "csi.storage.k8s.io/pod.uid":
+			continue
+		case "csi.storage.k8s.io/serviceaccount.name":
+			continue
```

Comment:
Would it be worth collapsing these into one case with commas?

Reply:
done